src/xenia/vfs/devices/disc_image_device.cc

```diff
@@ -127,9 +127,11 @@ bool DiscImageDevice::ReadEntry(ParseState* state, const uint8_t* buffer,
   entry->attributes_ = attributes | kFileAttributeReadOnly;
   entry->size_ = length;
   entry->allocation_size_ = xe::round_up(length, bytes_per_sector());
-  entry->create_timestamp_ = 0;
-  entry->access_timestamp_ = 0;
-  entry->write_timestamp_ = 0;
+
+  // Set to January 1, 1601 (UTC) in 100-nanosecond intervals
```

Comment:
Nope, it's 1601 for some strange reasons: [https://msdn.microsoft.com/en-us/library/windows/desktop/ms724284(v=vs.85).aspx](https://msdn.microsoft.com/en-us/library/windows/desktop/ms724284(v=vs.85).aspx)